proposed/2021/NuGetConsumePDBInPackageReference.md

```diff
@@ -0,0 +1,212 @@
+# Consuming pdb support with PackageReference
+
+* Status: Reviewing
+* Author(s): [Heng Liu](https://github.com/heng-liu)
+* Issue: [5926](https://github.com/NuGet/Home/issues/5926) Consume pdbs from packages in PackageReference - add Debug Symbols to LockFileTargetLibrary
+* Type: Feature
+
+## Problem Background
+
+In the PackageReference world, the packages are consumed from the global-packages folder, .pdb, .xml files from lib and runtime folder are not copied into the build output folder.
+However, in some cases, consuming .pbd, .xml files are needed to support certain features, like source link ( see issue [1458](https://github.com/dotnet/sdk/issues/1458)).
+But .NET SDK could not consume those files, as it needs [LockFileTargetLibrary](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTargetLibrary.cs) to tell what those asset files are, and NuGet doesn't have any reference to .pdb, .xml asset files in LockFileTargetLibrary for now.
```

Comment:
```
The established design for consuming files out of packages is that NuGet tells the .NET SDK which files are important though the assets file, or to be specific, the [LockFileTargetLibrary](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTargetLibrary.cs). 
Currently, NuGet does not select pdb or xml files.
```

Note that this isn't something you should have to accept, but just a general recommendation. 
It's a good idea to provide why we're doing this particular design :)

Reply:
Thanks! Fixed.